analyticsclient/client.py

```diff
@@ -63,17 +66,32 @@ def get(self, resource, timeout=None, data_format=DF.JSON):
         Raises: ClientError if the resource cannot be retrieved for any reason.
 
         """
-        response = self._request(resource, timeout=timeout, data_format=data_format)
+        return self._get_or_post(
+            self.METHOD_GET, resource,
+            timeout=timeout, data_format=data_format
+        )
```

Comment:
I like that pattern and agree that it's easier to read.  I wonder if there's some sort of pylint rule that we can turn on for this, and if not, we should record that style preference somewhere.

Anyway, I'd be happy to follow that pattern if the rest of the team agrees especially.